Yosemite/YosemiteTests/Stores/ProductCategoryStoreTests.swift

```diff
@@ -0,0 +1,109 @@
+import XCTest
+@testable import Yosemite
+@testable import Networking
+
+
+/// ProductCategoryStore Unit Tests
+///
+final class ProductCategoryStoreTests: XCTestCase {
+
+    /// Mockup Dispatcher!
+    ///
+    private var dispatcher: Dispatcher!
+
+    /// Mockup Storage: InMemory
+    ///
+    private var storageManager: MockupStorageManager!
+
+    /// Mockup Network: Allows us to inject predefined responses!
+    ///
+    private var network: MockupNetwork!
+
+    /// Store
+    ///
+    private var store: ProductCategoryStore!
+
+    /// Testing SiteID
+    ///
+    private let sampleSiteID: Int64 = 123
+
+    /// Testing Page Number
+    ///
+    private let defaultPageNumber = 1
+
+    /// Testing Page Size
+    ///
+    private let defaultPageSize = 75
+
+    // MARK: - Overridden Methods
+
+    override func setUp() {
+        super.setUp()
+        dispatcher = Dispatcher()
+        storageManager = MockupStorageManager()
```

Comment:
There doesn't seem to be a need to keep the `dispatcher` and `storageManage` as ivars. They're only used in the `ProductCategoryStore` below.

Reply:
Fixed https://github.com/woocommerce/woocommerce-ios/pull/2059/commits/8428b508f89d2ec59eeb86ccf89c7b9aca37c222